pages/how-we-work.md

```diff
@@ -26,7 +26,7 @@ We will work with your team to:
 - Draft your procurement documents or quickly review your drafts
 - Guide you through the vendor evaluation process to ensure the vendor is qualified to do the work 
 
-**[Learn more about how we can help](https://18f.gsa.gov/assets/presentations/18f-crisis-support-it-acquisitions.pdf)**
+**[Learn more about how we can help (PDF).](https://18f.gsa.gov/assets/presentations/18f-crisis-support-it-acquisitions.pdf)**
```

Comment:
This works, but we might want to consider putting an attribute selector in our CSS so we can capture all of these automatically, similar to how we insert the :lock: on links to private assets in the handbook.

Double-check me, but I think something like this should work: 

```a[href*=".pdf"]:after {
  content: " (PDF)";
}```